httpserver/service_node.h

```diff
@@ -14,6 +14,8 @@
 #include "swarm.h"
 
 static constexpr uint16_t SNODE_PORT = 8080;
+static constexpr auto LOKI_SENDER_SNODE_PUBKEY = "X-Loki-Snode-PubKey";
```

Comment:
These two probably belong in `http_connection.h`, and I would include "HEADER" somewhere in their names.

Reply:
Changed.